core/shared/platform/esp-idf/espidf_memmap.c

```diff
@@ -19,8 +19,13 @@ static portMUX_TYPE s_spinlock = portMUX_INITIALIZER_UNLOCKED;
 #endif
 
 void *
-os_mmap(void *hint, size_t size, int prot, int flags)
+os_mmap(void *hint, size_t size, int prot, int flags, os_file_handle file)
 {
+    if (os_is_handle_valid(&file)) {
```

Comment:
Seems that os_is_handle_valid isn't implemented in espidf platform

Reply:
Should we implement `os_is_handle_valid()` for these platforms ?

Reply:
Not necessarily I think, just let someone implement it when it is actually needed?

Reply:
OK